curriculum/js2/3.js

```diff
@@ -5,8 +5,17 @@
  * @returns {array}
  */
 
+const isPrime = ((num, i=2) => {
+  if (num < 2) return false;
+  if (i === num) return true;
+  if (num % i === 0) return false;
+  return isPrime(num, i+1);
+})
+
 const solution = (a) => {
-  return [];
+  return a.map(el => {
```

Comment:
`a.map(el => isPrime(el) ? el : 1 ) ` -> If you dont put `{`, javascript puts in a return by default to make code more concise.